frontend/common/constants.ts

```diff
@@ -213,6 +213,12 @@ export default {
     primary: '#6837fc',
     white: '#ffffff',
   },
+  contentTypes: {
+    environment: 30,
+    flag: 39,
+    project: 27,
+    segment: 55,
+  },
```

Comment:
This will vary per environment! We'll need to obtain this dynamically. There's an endpoint at `/api/v1/organisations/{organisation_pk}/metadata-model-fields/supported-content-types/` which looks like it does exactly this?

Reply:
ok, I am already using that endpoint to obtain that data